docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
+services:
+  postgres:
+    image: postgres:9.3
+
+  possum:
+    image: registry2.itci.conjur.net/possum:0.1.0-stable
+    command: server -a cucumber
+    environment:
+      DATABASE_URL: postgres://postgres@postgres/postgres
+      POSSUM_DATA_KEY: 'WMfApcDBtocRWV+ZSUP3Tjr5XNU+Z2FdBb6BEezejIs='
```

Comment:
Doesn't feel great to have the `POSSUM_DATA_KEY` set in the compose file, even for testing. Can you change this so it gets generated before bringing up possum, please?